src/components/ContactForm.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+
+const ContactForm = () => {
+  const [formData, setFormData] = useState({
+    name: '',
+    email: '',
+    message: '',
+  });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setFormData({
+      ...formData,
+      [name]: value,
+    });
```

Comment:
when modifying react state based on the old state it is often beneficial to provide a function for the setter.

This way the old version of the state is more likely to behave as expected

```suggestion
    setFormData((old) => ({
      ...old,
      [name]: value,
    }));
```